packages/elements-react/src/theme/default/components/settings/settings-totp.tsx

```diff
@@ -90,17 +93,25 @@ export function DefaultSettingsTotp({
             node={totpSecret}
             attributes={totpSecret.attributes as UiNodeInputAttributes}
           >
-            <Node.Input
-              node={totpSecret}
-              attributes={{
-                disabled: true,
-                name: "totp_secret_key",
-                node_type: "input",
-                type: "text",
-                value: (totpSecret.attributes as UiNodeTextAttributes).text
-                  .text,
-              }}
-            />
+            <div className="relative flex justify-stretch max-w-[488px]">
+              <input
+                disabled
+                name="totp_secret_key"
+                type="text"
+                value={
+                  (totpSecret.attributes as UiNodeTextAttributes).text.text
```

Comment:
This input field is controlled / fixed value, <Node.Input> is a much more complex component.